File: LeetCode/1041.Robot_Bounded_In_Circle/main.py
#!/usr/bin/python

from solution import Solution

# Unit Test
t1 = "GGLLGG"
t2 = "GG"
t3 = "GL"
s = Solution()
if (s.isRobotBounded(t1)==True):
    print("Pass")
else:
    print("Test 1 Failed")
if (s.isRobotBounded(t2)==False):
    print("Pass")
else:
    print("Test 2 Failed")
if (s.isRobotBounded(t3)==True):
    print("Pass")
else:
    print("Test 3 Failed")

File: LeetCode/1041.Robot_Bounded_In_Circle/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    string t1 = "GGLLGG";
    string t2 = "GG";
    string t3 = "GL";
    if (s.isRobotBounded(t1) == true) { cout << "Pass" << endl;}
    else { cout << "Failed 1 - t1" << endl;}
    if (s.isRobotBounded(t2) == false) { cout << "Pass" << endl;}
    else { cout << "Failed 2 - t2" << endl;}
    if (s.isRobotBounded(t3) == true) { cout << "Pass" << endl;}
    else { cout << "Failed 3 - t3" << endl;}
    return 0;
}